Sources/Database/Database.swift

```diff
@@ -1070,6 +1070,45 @@ private struct _Client {
         ON "episode_progresses" ("episode_sequence", "user_id")
         """
       )))
+      .flatMap(const(execute(
+        """
+      CREATE SEQUENCE IF NOT EXISTS users_referral_code_sequence
+      """
+      )))
+      .flatMap(const(execute(
+        """
+      CREATE OR REPLACE FUNCTION next_referral_code (OUT result bigint)
```

Comment:
This function uses the "snowflake" approach to unique id generation for shorter identifiers. We can take the resulting integer and render it in, e.g., base-36: `"c1vvvvjox"`

Reply:
(It's probably overkill, lol, but I didn't have any other approaches that came to mind.)